LiftSystem.esp/.gitignore

```diff
@@ -3,60 +3,3 @@
 .vscode/c_cpp_properties.json
 .vscode/launch.json
 .vscode/ipch
-
-src/ForeignDestinationSensor.cc
-src/ForeignDestinationSensor.hh
-src/ForeignEndstop.cc
-src/ForeignEndstop.hh
-src/ForeignLog.cc
-src/ForeignLog.hh
-src/ForeignMotor.cc
-src/ForeignMotor.hh
-src/ForeignPlanner.cc
-src/ForeignPlanner.hh
-src/ForeignPositionSensor.cc
-src/ForeignPositionSensor.hh
-src/ForeignTimer.cc
-src/ForeignTimer.hh
-src/HomeDown.cc
-src/HomeDown.hh
-src/HomeUp.cc
-src/HomeUp.hh
-src/IDestinationSensor.cc
-src/IDestinationSensor.hh
-src/IEndstop.cc
-src/IEndstop.hh
```

Comment:
Wat maak je me nou?

Reply:
Deze files werden vroeger in de src directory gegenereerd maar nu niet meer. Dan hoeven ze daar ook niet meer geignored te worden. Ofwel: deze wijziging is correct en zou geapproved moeten worden.